tests/unit/data/foo/ec2/2010-01-01/service-2.json

```diff
@@ -0,0 +1 @@
+{"foo": "bar"}
```

Comment:
For these new service files, is there something particular we're trying to validate with each?